src/zope/app/wsgi/testlayer.py

```diff
@@ -152,6 +152,9 @@ class FakeResponse(object):
 
     """
 
+    # XXX: zope.app.testing.functional used to respond with HTTP/1.1
```

Comment:
I am used to `XXX` meaning: This has to be fixed. Do you have any plans?

Reply:
What I would like to do is change this to hardcode b'HTTP/1.1'.

I don't know if there are people out there who use zope.app.wsgi and already expect HTTP/1.0 in their test suites.  If there are, my change might make them unhappy.

Reply:
Maybe we could do the change and raise the major version number.